client/index.html

```diff
@@ -301,20 +301,36 @@ <h1 class="title">Settings</h1>
 								</label>
 							</div>
 							{{/if}}
+							{{#unless public}}
+							<div class="col-sm-12">
+								<h2>Push Notifications</h2>
```

Comment:
We should consolidate "Push Notifications" and "Browser Notifications" into one group called "Notifications". From the user perspective, it just means being notified when not looking at the app, whatever strategy used underneath.

You mention both can be enabled on desktop, what is the use case for this?
I would expect that when app is opened (on browser), desktop notifications apply, but when it's shut down, push notifications are in use. The "Enable notifications" checkbox should enable them all.

If you have details as of what works in what context (push vs. normal notifications), that would help me figure out what would be the most usable thing to do :)